client/src/components/Sidebar/Sidebar.module.scss

```
@@ -26,3 +26,9 @@ $width: 215px;
   font-size: 0.75em; /* ~20px (16px design * 1.2 design-to-app ratio) */
   font-weight: 500;
 }
+
+.feedback-nav-item {
+  align-self: center;
+  margin-top: auto;
+  margin-bottom: 5px;
```

Could use a bit more space from the bottom
```suggestion
  margin-bottom: 1em;
```